stdlib/Random/src/Random.jl

```diff
@@ -14,6 +14,8 @@ using .DSFMT
 using Base.GMP.MPZ
 using Base.GMP: Limb
 
+import SHA
```

Comment:
Is this needed?

Reply:
As `SHA` is accessed, I guess yes? I prefer using qualifed names like `SHA.update!` here. Or did I misunderstand the question?

Reply:
Ok, I see: it's used in `RNG.jl` but not in this file. Might be clearer to move the `import SHA` into the file that uses it, but this is valid as well.

Reply:
Oh ok I understand. it's just that `Random.jl` already happens to have all the import/using directives necessary for the other files it includes.